packages/showcases/web-components/stories/components/vtmn-link/vtmn-link.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { html } from 'lit-html';
+import { spread } from '@open-wc/lit-helpers';
+
+export default {
+  title: 'Components/Link',
```

Comment:
```suggestion
  title: 'Components/vtmn-link',
```
And you can also change it for Button if you want :)

Reply:
As it is now shared in the core folder, how to name it ? VtmnLink or vtmn-link ?

Currently, it is labelled as VtmnLink for information.